plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                        description:
+                        - Refers to the logical segmentation of the network, grouping devices into isolated virtual networks.
+                        - Either virtual_network_name or vlan_id is mandatory for all operations in L3 Handoff with IP transit.
+                        type: str
+                      vlan_id:
+                        description:
+                        - Unique identifier assigned to a Virtual Local Area Network
+                        - Should be unique across the entire fabric site settings.
+                        - vlan_id can be from 1 to 4094. Except 1, 1002-1005, 2046, 4094.
+                        - Either virtual_network_name or vlan_id is mandatory for all operations in L3 Handoff with IP transit.
+                        - vlan_id cannot be updated.
+                        type: int
+                      tcp_mss_adjustment:
+                        description:
+                        - Allows the modification of the Maximum Segment Size in TCP packets.
+                        - tcp_mss_adjustment can be from 500 to 1440.
+                        - tcp_mss_adjustment can be updated.
+                        type: int
+                      local_ip_address:
+                        description:
+                        - IP address assigned to a devices interface within the fabric.
+                        - local_ip_address is for IPv4.
+                        - local_ip_address and remote_ip_address should fall in same subnet.
+                        - Either local and remote addresses or external_connectivity_ip_pool_name are mandatory.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        type: str
+                      remote_ip_address:
+                        description:
+                        - IP address of a device located outside the fabric network, often used for BGP peering.
+                        - remote_ip_address is for IPv4.
+                        - local_ip_address and remote_ip_address should fall in same subnet.
+                        - Either local and remote addresses or external_connectivity_ip_pool_name are mandatory.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        type: str
+                      local_ipv6_address:
+                        description:
+                        - IP address of a device located outside the fabric network, often used for BGP peering.
+                        - local_ipv6_address is for IPv6.
+                        - local_ipv6_address and remote_ipv6_address should fall in same subnet.
+                        - If remote_ipv6_address is provided, then local_ipv6_address is mandatory.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
```

Comment:
The below two statements are not meaningful. Can you check ? We are saying that local_ipv6_address and remote_ipv6_address are required, but if both are provided, then `external_connectivity_ip_pool_name` takes precedence... 

                    - If local_ipv6_address is provided, then remote_ipv6_address is required.
                    - If both are provided, `external_connectivity_ip_pool_name` takes precedence.
                    type: str